examples/mini-compile/test/MiniCompileTest.hs

```diff
@@ -82,6 +82,7 @@ data KindRep = KindRepTyConApp TyCon [KindRep]
 
 data TypeLitSort = TypeLitSymbol
                  | TypeLitNat
+                 | TypeLitChar
```

Comment:
maybe worth thinking about testing this on CI so we notice it breaks next time.

Reply:
yeah. we need an improvement. expect testing would have caught this. i'll raise an issue.

Reply:
https://github.com/digital-asset/ghc-lib/issues/278